source/MRMesh/MRObjectDistanceMap.cpp

```diff
@@ -168,8 +168,8 @@ tl::expected<void, std::string> ObjectDistanceMap::deserializeModel_( const std:
     if ( !res.has_value() )
         return tl::make_unexpected( res.error() );
     
-    dmap_ = std::make_shared<DistanceMap>( res.value() );
-
+    dmap_ = std::make_shared<DistanceMap>( res.value().first );
+    toWorldParams_ = res.value().second;
```

Comment:
should we still serialize and deserialize `toWorldParams_` as fields? (also this changes in Object serialization should support old files versions, so mb it is better to have old version here, and comlitelu new binnary format like `.mrdistancemap`)